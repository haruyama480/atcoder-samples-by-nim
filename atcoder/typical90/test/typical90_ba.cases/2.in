1
8
1
0
3
4
5
