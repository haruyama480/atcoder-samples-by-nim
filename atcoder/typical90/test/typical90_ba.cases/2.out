? 5
? 8
? 3
? 2
? 1
! 5
